src/main/java/com/treasure/hunt/view/settings/SettingsController.java

```diff
@@ -56,6 +56,8 @@ private static void addDescription(VBox vBox, Field field) {
 
         if (!description.isEmpty()) {
             Label descriptionLabel = new Label(description);
+            descriptionLabel.setMinHeight(Double.NEGATIVE_INFINITY);
+            descriptionLabel.setWrapText(true);
             descriptionLabel.getStyleClass().add("description");
             vBox.getChildren().add(descriptionLabel);
```

Comment:
So  @jotoh98 this shold also be defined in the fxml but I guess it is too late now

Reply:
I'd just turn it off then

Reply:
These are generated dynamically depending on fields in class Settings and their annotations. If wanted, I could implement one label as fxml and load it every time, but it seems a bit unnecessary considering the upper part of the corresponding setting block is changed for each type of field.